src/Test/Utilities/Shared/Assert/ConditionalFactAttribute.cs

```diff
@@ -37,4 +40,12 @@ public class HasShiftJisDefaultEncoding : ExecutionCondition
 
         public override string SkipReason => "OS default codepage is not Shift-JIS (932).";
     }
+
+    public class IsEnglishLocal : ExecutionCondition
+    {
+        public override bool ShouldSkip => 
+                System.Globalization.CultureInfo.CurrentCulture == new System.Globalization.CultureInfo("en-US");
```

Comment:
Clearly when I wrote "I'm not certain this is correct yet" I should have said "I hurried to get out the door to get to my bus, so I haven't re-read any of the code that I wrote".